api/client/service/opts.go

```diff
@@ -545,6 +548,7 @@ const (
 	flagRestartWindow      = "restart-window"
 	flagStopGracePeriod    = "stop-grace-period"
 	flagUpdateDelay        = "update-delay"
+	flagUpdateOnFailure    = "update-on-failure"
```

Comment:
It was originally `update-failure-action` in swarmkit. I changed it based on this comment: https://github.com/docker/swarmkit/pull/1204#discussion_r71886182 :)

I'll change it back to `update-failure-action`.